tests/RequestTest.php

```diff
@@ -1,18 +1,17 @@
 <?php
-namespace JoindinTest\Inc;
+namespace Joindin\Api\Test;
 
+use Joindin\Api\Request;
 use PHPUnit\Framework\TestCase;
 
-require_once __DIR__ . '/../../src/inc/Request.php';
-
 class RequestTest extends TestCase
 {
     /**
      * Make sure we have everything we need - in this case the config
      */
     public function setUp()
     {
-        include __DIR__ . '/../../src/config.php';
+        include __DIR__ . '/../src/config.php';
```

Comment:
`dirname()` would give you the parent path and `dirname(__DIR__)` would give you the absolute path of the parent directory of the current file's directory. So perhaps this might be a little cleaner:

```suggestion
        include dirname(__DIR__) . 'src/config.php';
```

Reply:
I'd much rather turn config.php into a configuration class, which would be even cleaner, however as this pr is big enough as it is I would like tot do those optimisations after this one